apps/web/src/components/StatusItem/StatusItem.tsx

```diff
@@ -146,7 +160,9 @@ const StatusItem = ({
   let effectiveIconColor = iconColor;
   switch (status) {
     case "optional":
-      effectiveIconColor = "primary";
+    case "not done":
+    case "locked":
+      effectiveIconColor = "black";
```

Comment:
This change seems to be trickling elsewhere like the below. Desired? 

<img width="1654" height="320" alt="image" src="https://github.com/user-attachments/assets/d4d9755c-aca2-4931-96b3-f65adc68da5d" />